src/main/java/lotto/domain/LottoGenerator.java

```diff
@@ -0,0 +1,40 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class LottoGenerator {
+
+    private static final int MAXIMUM_NUMBER = 45;
+    private static final int LOTTO_NUMBER_LIMIT = 6;
+    private static final String DELIMITER = ",";
+    private final List<Integer> numbers = new ArrayList<>();
+
+    public LottoGenerator() {
+        for (int i = 1; i <= MAXIMUM_NUMBER; i++) {
+            numbers.add(i);
+        }
+    }
+
+    public Set<LottoNumber> generateAuto() {
+        List<Integer> lottoNumbers = new ArrayList<>(numbers);
+        Collections.shuffle(lottoNumbers);
+        lottoNumbers = lottoNumbers.subList(0, LOTTO_NUMBER_LIMIT);
+        return lottoNumbers.stream()
+            .map(LottoNumber::of)
+            .collect(Collectors.toSet());
+    }
```

Comment:
numbers를 미리 생성해둔다면 
```suggestion
    public Set<LottoNumber> generateAuto() {
        Collections.shuffle(numbers);
        List<Integer> lottoNumbers = numbers.subList(0, LOTTO_NUMBER_LIMIT);
        return lottoNumbers.stream()
            .map(LottoNumber::of)
            .collect(Collectors.toSet());
    }
```
불필요한 객체 생성도 줄일 수 있을거 같아요! 그리고 LottoGenerator인데 generate()했을때 Lotto를 반환해 주는 것은 어떨까요??